src/util/dns/add-dns-record.ts

```diff
@@ -18,6 +18,10 @@ export default async function addDNSRecord(
     });
     return record;
   } catch (error) {
+    if (error.status === 400) {
+      return error;
```

Comment:
I think that's wrong because the error has an `error` object so later on reading from `message` will result to `undefined`. Probably it's best if we model a `AddDNSRecordError` and we pass the message or the payload so we still print the text that comes from API

Reply:
That's not correct. `error` object itself contains `message` and `status`. Why should we create a special type for every possible API error if the only thing we usually want to do is to extract `error.message`?